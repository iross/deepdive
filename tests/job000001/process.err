  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100 45664  100 45664    0     0  1773k      0 --:--:-- --:--:-- --:--:-- 1938k
