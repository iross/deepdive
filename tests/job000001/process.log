000 (132501.000.000) 12/29 15:10:43 Job submitted from host: <128.104.101.254:9618?noUDP&sock=4243_951b_3>
    DAG Node: job000001+job000001
...
001 (132501.000.000) 12/29 15:13:52 Job executing on host: <128.105.245.105:39124>
...
006 (132501.000.000) 12/29 15:14:00 Image size of job updated: 131604
	11  -  MemoryUsage of job (MB)
	10328  -  ResidentSetSize of job (KB)
...
006 (132501.000.000) 12/29 15:19:00 Image size of job updated: 362244
	220  -  MemoryUsage of job (MB)
	224660  -  ResidentSetSize of job (KB)
...
006 (132501.000.000) 12/29 15:23:01 Image size of job updated: 368796
	220  -  MemoryUsage of job (MB)
	224660  -  ResidentSetSize of job (KB)
...
005 (132501.000.000) 12/29 15:23:01 Job terminated.
	(1) Normal termination (return value 8)
		Usr 0 00:04:33, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:04:33, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	519748  -  Run Bytes Sent By Job
	1927656  -  Run Bytes Received By Job
	519748  -  Total Bytes Sent By Job
	1927656  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :   678215  1000000   3153624
	   Memory (MB)          :      220     1000      1000
...
