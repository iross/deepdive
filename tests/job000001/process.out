_CONDOR_ANCESTOR_5450=10477:1418712817:1736197032
_CONDOR_ANCESTOR_10477=12122:1419891701:1359493938
_CONDOR_ANCESTOR_12122=12127:1419891702:3631151518
TEMP=/var/lib/condor/execute/slot1/dir_12122
_CONDOR_SCRATCH_DIR=/var/lib/condor/execute/slot1/dir_12122
_CONDOR_SLOT=slot1_12
BATCH_SYSTEM=HTCondor
TMPDIR=/var/lib/condor/execute/slot1/dir_12122
_CONDOR_CHIRP_CONFIG=/var/lib/condor/execute/slot1/dir_12122/.chirp.config
_CONDOR_JOB_PIDS=
TMP=/var/lib/condor/execute/slot1/dir_12122
_CONDOR_JOB_AD=/var/lib/condor/execute/slot1/dir_12122/.job.ad
_CONDOR_JOB_IWD=/var/lib/condor/execute/slot1/dir_12122
_CHIRP_DELAYED_UPDATE_PREFIX=Chirp*
_CONDOR_MACHINE_AD=/var/lib/condor/execute/slot1/dir_12122/.machine.ad
PATH=/usr/local/bin:/usr/kerberos/bin:/usr/bin:/bin
MachineAdTest: looking for <IsEuclid>
Euclid test returned <no>
Calling curl to fetch chtcinnerwrapper try 0
Curl error <0>
MachineAdTest: looking for <IsEuclid>
Done Running chtcinnerwrapper Result was <0>
All went well!
